setup.py

```diff
@@ -1,4 +1,3 @@
-#!/usr/bin/python
```

Comment:
I can't recall any documentation for a Python project saying to run `./setup.py install`, but I already put the shebang back in :)